test/runtests.jl

```diff
@@ -1129,3 +1129,77 @@ end
         @test coef(glm_dense_alt) ≈ coef(glm_views_alt)
     end
 end
+
+@testset "PowerLink" begin
+    @testset "Functions related to PowerLink" begin
+        @test GLM.linkfun(IdentityLink(), 10) ≈ GLM.linkfun(PowerLink(1), 10)
+        @test GLM.linkfun(SqrtLink(), 10) ≈ GLM.linkfun(PowerLink(0.5), 10)
+        @test GLM.linkfun(LogLink(), 10) ≈ GLM.linkfun(PowerLink(0), 10)
+        @test GLM.linkfun(InverseLink(), 10) ≈ GLM.linkfun(PowerLink(-1), 10)
+        @test GLM.linkfun(InverseSquareLink(), 10) ≈ GLM.linkfun(PowerLink(-2), 10)
+        @test GLM.linkfun(PowerLink(1 / 3), 10) ≈ 2.154434690031884
+
+        @test GLM.linkinv(IdentityLink(), 10) ≈ GLM.linkinv(PowerLink(1), 10)
+        @test GLM.linkinv(SqrtLink(), 10) ≈ GLM.linkinv(PowerLink(0.5), 10)
+        @test GLM.linkinv(LogLink(), 10) ≈ GLM.linkinv(PowerLink(0), 10)
+        @test GLM.linkinv(InverseLink(), 10) ≈ GLM.linkinv(PowerLink(-1), 10)
+        @test GLM.linkinv(InverseSquareLink(), 10) ≈ GLM.linkinv(PowerLink(-2), 10)
+        @test GLM.linkinv(PowerLink(1 / 3), 10) ≈ 1000.0
+
+        @test GLM.mueta(IdentityLink(), 10) ≈ GLM.mueta(PowerLink(1), 10)
+        @test GLM.mueta(SqrtLink(), 10) ≈ GLM.mueta(PowerLink(0.5), 10)
+        @test GLM.mueta(LogLink(), 10) ≈ GLM.mueta(PowerLink(0), 10)
+        @test GLM.mueta(InverseLink(), 10) ≈ GLM.mueta(PowerLink(-1), 10)
+        @test GLM.mueta(InverseSquareLink(), 10) == GLM.mueta(PowerLink(-2), 10)
+        @test GLM.mueta(PowerLink(1 / 3), 10) ≈ 300.0
+
+        @test PowerLink(1 / 3) == PowerLink(1 / 3)
+        @test isequal(PowerLink(1 / 3), PowerLink(1 / 3))
+        @test !isequal(PowerLink(1 / 3), PowerLink(0.33))
+        @test hash(PowerLink(1 / 3)) == hash(PowerLink(1 / 3))
+    end
+    trees = dataset("datasets", "trees")
+    @testset "GLM with PowerLink" begin
+        mdl = glm(@formula(Volume ~ Height + Girth), trees, Normal(), PowerLink(1 / 3))
+        @test coef(mdl) ≈ [-0.051323413605527, 0.014286863108177, 0.150331244093515]
+        @test stderror(mdl) ≈ [0.22409428535, 0.00334243140, 0.00583824078] atol=1.0e-5
+        @test confint(mdl) ≈ [-0.492662575350799992 0.38142669625130310;
+        0.007782050247944785 0.02082315251811211; 0.139020340418155641 0.16184256030515429] atol=1.0e-2
+        @test dof(mdl) == 4
+        @test GLM.dispersion(mdl.model, true) ≈ 6.577133411105055 atol=1.0e-4
+        @test loglikelihood(mdl) ≈ -71.60507986988925
+        @test deviance(mdl) ≈ 184.15774688122
+        @test aic(mdl) ≈ 151.21015973978
+        @test predict(mdl)[1] ≈ 10.59735242595150
+    end
+    @testset "Compare PowerLink(0) and LogLink" begin
+        mdl1 = glm(@formula(Volume ~ Height + Girth), trees, Normal(), PowerLink(0))
+        mdl2 = glm(@formula(Volume ~ Height + Girth), trees, Normal(), LogLink())
+        @test coef(mdl1) ≈ coef(mdl2)
+        @test stderror(mdl1) ≈ stderror(mdl2)
+        @test confint(mdl1) ≈ confint(mdl2)
+        @test aic(mdl1) ≈ aic(mdl2)
```

Comment:
Can you compare `loglikelihood`, `deviance` and `dof` rather than `aic`? The latter is defined based on the former anyway.

Reply:
We have added.